src/pages/baan-profile.tsx

```diff
@@ -0,0 +1,235 @@
+import Image from 'next/image';
+
+const profilePic = '/images/pfp-placeholder.svg';
+
+const BaanProfile = () => {
+    const bgStyle: string = 'lg:mb-none lg:mb-6 p-auto py-8 px-8';
+    const bottonStyle: string =
+        'lg:mx-4 lg:px-6 text-sm py-2 mx-auto px-3 rounded-lg ring-4 ring-gray-300';
+    interface GroupMember {
```

Comment:
We already have an `IUser `interface available in `src/types/user.ts.` Please utilize that instead.